### ISSUE TITLE
Request for more information / new step-by-step guide - see: https://bugzilla.suse.com/show_bug.cgi?id=1216632

### ISSUE THREAD

**Issue by chabowski** [SUSE/suse-best-practices#407]
@fmherschel @lpinne could you please have a look at the following - looks like a request for an additional SBP guide:

Bug https://bugzilla.suse.com/show_bug.cgi?id=1216632 has been submitted by Ursula Brueckner @ubrueck and refers to:

https://documentation.suse.com/sbp/sap-15/html/SAP-nw740-sle15-setupguide/index.html

-----------------------------------------------------------------

Customer was migrating from IBM V9000 storage to V9500 storage and was looking for documentation how to replace the shared disk drive they setup according to:

https://documentation.suse.com/sbp/all/html/SAP-nw740-sle15-setupguide/index.html

3.5.1 Shared disk for cluster ASCS and ERS

```
[...]
partition one (/dev/sdb1) for SBD (7M)
partition two (/dev/sdb2) for the first file system (10GB) formatted with XFS
partition three (/dev/sdb3) for the second file system (10GB) formatted with XFS
[...]
```

(Non volume group LVM controlled by HAE,
and they needed to replace  the disk for that file system on both nodes as this disk is shared across 2 nodes and one partition is mounted on each node and they had the SBD volume on the same disk.)

The recommended options for SBD to initialize a new SBD disk and reconfigure the paths in pacemaker and additional notes to backup fs for ASCS-ERS, mount the new volume under the same mount point and restore the fs from backup wasn't as helpful as they'd expected.

They were looking for a step-by-step Guide on the procedure? 
Maybe this can be written/documented under our "SUSE Best Practices" guides?

On a side note:
For ASCS-ERS under SLES/SAP 15 it's possible to use a Simple-Mount setup so they won't need shared disks for the SAP instances anymore only for SBD.

Our documentation covers the Simple-Mount setup:
- https://www.suse.com/c/simple-mount-structure-for-sap-application-platform/
- https://www.suse.com/c/first-certified-simple-mount-cluster-ever/
- https://documentation.suse.com/sbp/sap/single-html/SAP-S4HA10-setupguide-simplemount-sle15/

**Comment by lpinne** [comment]
Hi,

1) This guide is in maintenance mode. It should not be used for new deployments. See
https://documentation.suse.com/sbp/sap-15/html/SAP-nw740-sle15-setupguide/index.html#id-introduction
---
For deploying new HA clusters, we recommend using the so-called simple mount setup described in SUSE TID "Use of Filesystem resource for ASCS/ERS HA setup not possible" (https://www.suse.com/support/kb/doc/?id=000019944). 
---

2) The SAP HA setup guides are focussing on SAP specific topics. The mentioned shared storage layers are the same as for other workloads. The generic SUSE HA docu and the legacy Simple Stack setup guide might be used. See
https://documentation.suse.com/sbp/sap-15/html/SAP-nw740-sle15-setupguide/index.html#id-shared-disk-for-cluster-ascs-and-ers
---
However, you can also place these file systems in separate volume groups. In that case, you need to add further cluster resources to control the logical volume groups. This is out of the scope of this setup guide.
---

From that two point, I would suggest to close this issue.

Regards,
Lars

**Comment by lpinne** [comment]
see comment above

**Comment by chabowski** [comment]
Thank you Lars! Okay - just added Ursula so that she has all the comments and information - I will close the issue again tomorrow

**Comment by chabowski** [comment]
@ubrueck Hi Ursula - just have a look at Lars' comments, please. Thank you!

**Comment by ubrueck** [comment]
Yes, for new deployments they can use simple mount setup (I recommended it to them for future deployments) or place them on separate VGs. However this was for a customer's already existing setup. I'm assuming there are more who deployed according to the old setup guide and eventually replace their shared drive.
Their request was two-sided, they were also missing a step-by-step guide how to replace/ initialize a new SBD disk.

**Comment by lpinne** [comment]
Hi ubrueck,

thank you for giving background info!

1) Migrating a shared filesystem based setup to a simple mount one is out of scope for the setup guides.
If there is demand, SUSE might publish a dedicated guide or TID. Big challenge wil be the variations of
existing filesystem based deployments. Maybe only the NFS-based setup could be safely covered. 
In any case, professional services would be glad to help customers with the migration.

2) The questions on changing SBD might go to the generic HA docu. This is not specific to SAP workloads.
It is a quite common question and has been raised several times.
Maybe TIDs  7021158, 7010933, 7018194, 7007616, 7023689 contain helpful infos on changing SBDs.

Please understand that we can not cover everything HA related within the HA for SAP setup guides.
Regards,
Lars
